enterprise/backend/src/metabase_enterprise/metabot_v3/dummy_tools.clj

```diff
@@ -0,0 +1,75 @@
+(ns metabase-enterprise.metabot-v3.dummy-tools
+  (:require
+   [cheshire.core :as json]
+   [metabase-enterprise.metabot-v3.tools.create-dashboard-subscription]
+   [metabase-enterprise.metabot-v3.tools.query]
+   [metabase-enterprise.metabot-v3.tools.who-is-your-favorite]
+   [metabase.api.common :as api]
+   [toucan2.core :as t2]))
+
+(defn- get-current-user
+  [_ _ context]
+  {:output (if-let [{:keys [id email first_name last_name]}
+                    (or (some-> api/*current-user* deref)
+                        (t2/select-one [:model/User :id :email :first_name :last_name] api/*current-user-id*))]
+             {:user_id id
+              :name (str first_name " " last_name)
+              :email_address email}
+             {:error "current user not found"})
+   :context context})
+
+(defn- get-dashboard-details
+  [_ {:keys [dashboard_id]} context]
+  {:output (or (t2/select-one [:model/Dashboard :id :description :name] dashboard_id)
+               {:error "dashboard not found"})
+   :context context})
```

Comment:
`get-current-user` and `get-dashboard-details` implement the current tool interface. The parameters are the name of the tool, the arguments, and the context. My current understanding is that tools should not need or change the context and this interface can be simplified.

I haven't registered them as "real" tools.